.jenkins/actions/build_bare_metal.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# stop on all errors
+set -e
+
+INSTALL_DIR=${PROJECT}/../install
+FV3GFSEXE_DIR=${INSTALL_DIR}/fv3gfs-fortran/
```

Comment:
This plan is intended to run on commits to master or on a cron but not on pull requests? I ask because installing this in the official install dir, where I presume people will be running it, could potentially cause issues. e.g. if it gets ripped out from under someone while they are running a performance benchmark...

Reply:
The intent was to run this weekly and since executables are copied in (so interrupt should be minimally invasive). But I agree that it's not ideal since even if this is rare, things can suddenly change with a new executable coming in. Not sure how to deal with this. I would also not want to retain a huge archive of executables under `/install/...`. It's the same dilemma as with the develop branch, I guess. What would you recommend @rheacangeo?